Dockerfile

```diff
@@ -5,16 +5,6 @@ ARG BUILD_NUMBER
 ARG GIT_REF
 ARG GIT_BRANCH
 
-LABEL maintainer="HMPPS Digital Studio <info@digital.justice.gov.uk>"
```

Comment:
These are no longer needed as they are included as part of the HMPPS base image.